js/pages/options.es6.js

```diff
@@ -14,6 +15,7 @@ function Options (ops) {
 
 Options.prototype = $.extend({},
     Parent.prototype,
+    mixins.setBrowserClassOnBodyTag,
```

Comment:
What's the advantage of adding this functionality as a mixin vs adding it to the parent class (Page), or as some utility function?